.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 .env
-test.ts
\ No newline at end of file
+test.ts
```

Comment:
Is this needed?

Reply:
I like coding along with the lessons, and I thought it does no harm if I have this on my local machine only. I think coding along with the lessons helps learning, and this file serves that purpose. So I like having it there. In your opinion, does it violate any principles?

Reply:
Nah feel free to leave it I was curious if it was intended or not